packages/odyssey-design-tokens/src/font.json

```diff
@@ -1,5 +1,28 @@
 {
   "font": {
+    "color": {
```

Comment:
Open to naming changes here. `body` and `heading` have clear use cases, not sure about the rest.

cc @gretchen-okta